etl-federation-aggregator/secondary-federated.yaml

```diff
@@ -0,0 +1,36 @@
+global:
+  prometheus:
+    enabled: true
+    # fqdn: http://prometheus-operated.monitoring:9090
+  grafana:
+    enabled: false
+    proxy: false
+kubecostProductConfigs:
+  clusterName: CLUSTER_NAME
+federatedETL:
+  federatedCluster: true
```

Comment:
agentOnly: true
will disable grafana, forecasting and frontend.
sorry! and thanks for doing this!